src/software/world/world.h

```diff
@@ -51,7 +52,7 @@ class World final
      *
      * @param new_ball_data A Ball containing new ball information
```

Comment:
Can we update the comment here to reflect that new_ball_data is a ball state, not a ball?

Reply:
updated